### PR TITLE
Feat/post content styles

### PR DESCRIPTION
**Pull request by smaraghi**
These are the base post content styles. 

At the bottom of the Test Post Components & Blocks post, I added several components to test these styles including a table, list elements, external links, etc. 

Please let me know if there is anything I missed.

### CODE REVIEW

File: wp-content/themes/reconasia/assets/_scss/pages/single.scss [hunk 1]
```diff
@@ -15,91 +15,170 @@
...
+      @extend %text-style-paragraph-medium-long;
       line-height: 1.4 !important;
     }
 
     li + li {
       margin-top: rem(8);
     }
+
+    ol::marker,
+    li::marker {
```
Comment (urchykli) [inline wp-content/themes/reconasia/assets/_scss/pages/single.scss:69]: Can we add some space between the markers and the `li`s please?

File: wp-content/themes/reconasia/assets/_scss/pages/single.scss [hunk 2]
```diff
@@ -15,91 +15,170 @@
...
-      // @include link-underline($color-primary-base);
+    &.aligncenter,
+    &.alignleft,
+    &.alignright {
+      width: 100%;
+    }
+
+    &.is-style-stripes {
+      thead {
+        background-color: $color-bg-thead;
```
Comment (urchykli) [inline wp-content/themes/reconasia/assets/_scss/pages/single.scss:123]:
```suggestion
        background-color: $color-bg-light-300;
```

File: wp-content/themes/reconasia/assets/_scss/pages/single.scss [hunk 3]
```diff
@@ -15,91 +15,170 @@
...
+      }
 
-      &:hover {
-        // color: $color-primary-base;
+      tbody {
+        tr:nth-child(even) {
+          background-color: $color-bg-tbody;
```
Comment (urchykli) [inline wp-content/themes/reconasia/assets/_scss/pages/single.scss:128]:
```suggestion
          background-color: $color-bg-light-200;
```

File: wp-content/themes/reconasia/assets/_scss/pages/single.scss [hunk 4]
```diff
@@ -15,91 +15,170 @@
...
+        tr:nth-child(odd) {
+          background-color: $color-white-100;
+        }
       }
 
-      &:visited {
-        // color: $color-text-gray-70;
+      table td + td,
+      table th + th {
+        border-left: 1px solid rgba(0, 0, 0, 0.15);
```
Comment (urchykli) [inline wp-content/themes/reconasia/assets/_scss/pages/single.scss:137]:
```suggestion
        border-left: 1px solid $color-black-115;
```

File: wp-content/themes/reconasia/assets/_scss/pages/single.scss [hunk 5]
```diff
@@ -15,91 +15,170 @@
...
+      }
+
+      td {
+        border-bottom: 1px solid rgba(0, 0, 0, 0.15);
```
Comment (urchykli) [inline wp-content/themes/reconasia/assets/_scss/pages/single.scss:141]:
```suggestion
        border-bottom: 1px solid $color-black-115;
```

File: wp-content/themes/reconasia/assets/_scss/pages/single.scss [hunk 6]
```diff
@@ -15,91 +15,170 @@
...
+      margin-bottom: rem(8px);
+      @extend %text-style-paragraph-large-short-sans;
+      border-collapse: collapse;
+    }
+
+    thead {
+      @extend %text-style-label-large-strong;
+
+      th {
+        border-bottom: 1px solid rgba(0, 0, 0, 0.3) !important;
```
Comment (urchykli) [inline wp-content/themes/reconasia/assets/_scss/pages/single.scss:157]:
```suggestion
        border-bottom: 1px solid $color-black-130 !important;
```

File: wp-content/themes/reconasia/assets/_scss/pages/single.scss [hunk 7]
```diff
@@ -15,91 +15,170 @@
...
+        text-align: left;
       }
     }
+
+    th,
+    td {
+      padding: rem(12);
+    }
+
+    figcaption {
```
Comment (urchykli) [inline wp-content/themes/reconasia/assets/_scss/pages/single.scss:167]: We need to add the bottom underline to the caption.

**Review by urchykli — CHANGES_REQUESTED** [review]
Thanks for helping out with this, Serven! In addition to some inline comments, I had the following notes. Please let me know if you have any questions.
- Let's remove the external link icon from the publication block title.
- We need inline link styles.  I added an inline link by the lists.
- We need to add font, etc for the external links.
- Focus styles for inline and external links.
- Is the external link icon supposed to be red on hover?
- We need to add a top border to the table.

File: wp-content/themes/reconasia/assets/_scss/abstracts/_variables.scss [hunk 8]
```diff
@@ -51,6 +51,9 @@ $color-border-dark-150: $color-black-150;
 
 $color-page-header-triangle: rgba(53, 62, 66, 1);
 
+$color-bg-thead: rgb(230, 233, 236);
+$color-bg-tbody: rgb(241, 243, 245);
```
[the remark this answers is not in the document]
Comment (smaraghi) [reply wp-content/themes/reconasia/assets/_scss/abstracts/_variables.scss:55]: The colors on figma were different rgbas which is why I added them. Let me know if you would like to forgo them

File: wp-content/themes/reconasia/assets/_scss/pages/single.scss [hunk 9]
```diff
@@ -15,91 +15,224 @@
...
+      text-decoration: underline;
+      text-transform: unset;
 
       &:hover {
-        // color: $color-primary-base;
+        color: $color-accent-light-primary-100;
+        text-decoration: none;
+      }
+
+      &:focus {
```
Comment (urchykli) [inline wp-content/themes/reconasia/assets/_scss/pages/single.scss:133]: We need to remove the underline on focus.

File: wp-content/themes/reconasia/assets/_scss/blocks/post/_publications.scss [hunk 10]
```diff
@@ -16,13 +16,18 @@
     &:hover {
       .csis-block__title,
```
[the remark this answers is not in the document]
Comment (smaraghi) [reply wp-content/themes/reconasia/assets/_scss/blocks/post/_publications.scss:17]: With:

image

Without:

image

File: wp-content/themes/reconasia/assets/_scss/blocks/post/_publications.scss [hunk 11]
```diff
@@ -56,11 +61,14 @@
       @extend %text-style-label-large-strong;
 
       a {
+        color: $color-black-190;
```
[the remark this answers is not in the document]
Comment (smaraghi) [reply wp-content/themes/reconasia/assets/_scss/blocks/post/_publications.scss:64]: With:

image


Without:

image

File: wp-content/themes/reconasia/assets/_scss/blocks/post/_publications.scss [hunk 12]
```diff
@@ -56,11 +61,14 @@
...
+        text-transform: unset;
         box-shadow: none;
+        @include font-size(18px);
```
[the remark this answers is not in the document]
Comment (smaraghi) [reply wp-content/themes/reconasia/assets/_scss/blocks/post/_publications.scss:67]: With:

image


Without:

image

[the remark this answers is not in the document]
Comment (smaraghi) [reply wp-content/themes/reconasia/assets/_scss/blocks/post/_publications.scss:67]: When its not on the a it is the wrong size - again another overwriting issue

**Pull request merged**